File: draft/src/music.js

// this is called when the backend sends any message
function runCommand(cmd) {
    if (cmd.function == 'play') {
        play(cmd.note);
    }
}

const synth = new Tone.Synth().toDestination();

function play(note) {
    synth.triggerAttackRelease(note, "8n")
}

File: draft/src/music.py
import json
from time import sleep

def sendmsg(msgtype, **cmd):
  """Sends a message to the frontend.

  The frontend will receive the specified message whenever
  this function is called. The frontend can decide to some
  action on each of these messages.
  """
  msg = dict(msgtype=msgtype, cmd=cmd)
  print("--MSG--", json.dumps(msg))

def play(note):
  sendmsg("music", function="play", note=note)